CHANGELOG.md

```diff
@@ -1,12 +1,80 @@
 # Changelog
 
-## Unreleased
+## v6.0.0-rc.2 - 2017-11-14
 
-- Update the contribution guideline to note that updating the changelog is required. (#5586)
+### Breaking changes
+
+- Skip null-element check for collections with a value-type element type (#5432)
+- Make 'AllSigned' execution policy require modules under $PSHome to be signed (#5511)
+
+### Engine updates and fixes
+
+- Update powershell to use `2.0.4` dotnet core runtime. (#5677)
+- Remove references to the old executable `powershell` or `powershell.exe` (#5408)
+
+### General cmdlet updates and fixes
+
+- Remove unncessary check for Paths.count > 0, in the `*-FileCatalog` CmdLets, as there is code later to use the current (#5596)
```

Comment:
The clause after the comma seems to end abruptly.  I don't think it's even needed at all.  Can just put a period after `Cmdlets`.

Reply:
resolved